docs/index.rst

```diff
@@ -142,8 +139,8 @@ The Yellowbrick `license <https://github.com/DistrictDataLabs/yellowbrick/blob/m
 Yellowbrick is hosted on `GitHub <https://github.com/DistrictDataLabs/yellowbrick/>`_. The `issues <https://github.com/DistrictDataLabs/yellowbrick/issues/>`_ and `pull requests <https://github.com/DistrictDataLabs/yellowbrick/pulls>`_ are tracked there.
 
 
-Documentation Table of Contents
----------------------------
```

Comment:
The whitespace and header underlines were causing errors in the documentation build, so I quickly updated them here.